makefile

```
@@ -40,16 +52,18 @@ compile-windows: clean-windows build-windows copyenv-windows
 	rm -rf ./cocktails.mcp/dist/linux && mkdir -p ./cocktails.mcp/dist/linux
 
-copyenv-linux:
+copyenv:
 	cp ./cocktails.mcp/src/.env.local ./cocktails.mcp/dist/linux/.env.local && cp ./cocktails.mcp/src/.env ./cocktails.mcp/dist/linux/.env
 
-build-linux:
+build:
 	cd ./cocktails.mcp/src && CGO_ENABLED=0 GOOS=linux go build -o ../dist/linux/cezzis-cocktails ./cmd
 
-compile-linux-ci: clean-linux build-linux
+compile: clean-linux build-linux copyenv-linux
```

checkmake (0.2.2)

[warning] Target "compile" should be declared PHONY.

(phonydeclared)